wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
Do you need this script in both the front-end and the back-end?

Reply:
Yep, because the process that uses this doesn't have access to the backend code. It is needed so that we don't start testing before both frontend _and_  backend are up and running.  
It could also be downloaded every time, but I don't think that's a better solution than having it in the repo(s).